awx/api/serializers.py

```diff
@@ -4453,6 +4454,122 @@ def validate(self, attrs):
         return accepted
 
 
+class BulkJobNodeSerializer(serializers.Serializer):
+    # if we can find out the user, we can filter down the UnifiedJobTemplate objects
+    unified_job_template = serializers.PrimaryKeyRelatedField(queryset=UnifiedJobTemplate.objects.all(), many=False)
+    inventory = serializers.PrimaryKeyRelatedField(queryset=Inventory.objects.all(), required=False, many=False)
+    credentials = serializers.PrimaryKeyRelatedField(queryset=Credential.objects.all(), required=False, many=True)
+    identifier = serializers.CharField(required=False, write_only=True, allow_blank=False)
+
+    class Meta:
+        fields = (
+            'unified_job_template',
+            'identifier',
+            'inventory',
+            'credentials',
+            #   'labels',
+            #   'extra_data',
+            #   'survey_passwords',
+            #   'char_prompts',
+            #   'labels',
+            #   'instance_groups',
+            #   'execution_environment',
+        )
+
+    def validate(self, attrs):
+        # Since we don't validate using the model here, I don't want to pass any extra things that we are not
+        # explicitly checking RBAC on up to the BulkJobsLaunchSerializer because we could allow people to launch with
+        # related items they don't have permission to
+        for key in attrs.keys():
+            if key not in self.fields:
+                attrs.pop(key)
```

Comment:
This isn't necessary, the `validated_data` dict that results from the validation process will contain only valid fields from `fields`.